components/docs-chef-io/content/automate/chef_automate_profile.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "User Profile"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "User Profile"
+    parent = "automate/settings"
+    identifier = "automate/settings/chef_automate_profile.md User Profile"
+    weight = 120
++++
+
+## Overview
+
+Chef Automate users can manage their display name, date/time format, and password. Select the user icon in the top navigation bar, and select **Profile** from the drop-down menu.
+
+{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}
+
+Select **Profile** to view the details. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you edit your password.
+
+### Change a Display Name
+
+Change your display name by entering your preferred name. The display name gets updated as soon as you change the name in the bar and click **Save**.
```

Comment:
```suggestion
Change your display name by entering your preferred name. Select **Save** to update your name.
```

Reply:
I'm concerned that my phrase `across Chef Automate.` is too colloquial. Does it make sense to you?